### DISCUSSION TITLE
Courseplay Harvester settings needs

### DISCUSSION THREAD

**Discussion by petersonfarm** [Courseplay/Courseplay_FS22#3049] [Ideas]
I would really like to see a setting like the old courseplay to manually call an unloader, stop the unloader, and a more accurate path for grain carts to get straight next to the wagons so when it backs up the auger doesnt leave the hopper causing it to stop unloading. FS17 was by far the best courseplay. small tweaks

**Comment by Tensuko** [comment]
Did not happen in FS19 and will not happen in FS22.
Will see if we will do anything about it in the next FS.

**Comment by petersonfarm** [comment]
Nobody said fs19 had it.


Sent from Yahoo Mail for iPhone


On Wednesday, January 3, 2024, 11:33 AM, Tensuko ***@***.***> wrote:




Did not happen in FS19 and will not happen in FS22.
Will see if we will do anything about it in the next FS.
